vercel.json

```diff
@@ -48,26 +48,6 @@
       "source": "/platforms/(.*)/configuration/install-cdn/",
       "destination": "/platforms/$1/install/cdn/"
     },
-    {
-      "source": "/platforms/(.*)/enriching-error-data/set-level/",
```

Comment:
pretty sure i had this redirect wrong and it was sitting under additional data, but its a minor page so doubt anyone would complain